howfairis/repo.py

```
@@ -89,13 +89,12 @@ def _derive_raw_url_format_string(self):
         return raw_url_format_string
 
     def _get_default_branch(self):
-        fallback_branch = 'main'
         # GitHub API and GitLab API work the same
```

i removed this because failing early is better